_entry.sh

```diff
@@ -1,8 +1,14 @@
 #!/bin/sh
 set -e
 
+if [ "$1" != "${1#-}" ]; then
+    # if the first argument is an option like `--help` or `-h`
+    exec deno "$@"
+fi
+
 case "$1" in
     bundle | cache | compile | completions | coverage | doc | eval | fmt | help | info | install | lint | lsp | repl | run | test | types | upgrade )
```

Comment:
I had thought I had a script to generate this (which this change might break)... but don't see it in the repo :(

Reply:
Hm... yeah. Having a script to automatically detect the possible commands (and maybe even options) would be good. But since we don't have it already, I think it's harmless to merge this first.